cmd/api/src/database/migration/stepwise_internal_integration_test.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2023 Specter Ops, Inc.
```

Comment:
I agree, and the reason we don't have both in the same filename is because it's _highly unusual_ to have an internal integration test. However, that's how this test was originally authored and my attempt to turn it into a proper non-internal integration test was going to be too messy for this PR. I'd like to keep it ugly both because it's more accurate (makes it obvious what kinds of tests it contains) as well as being a beacon for eventual refactor